developers/wcs/embeddings/administration.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Administration
+sidebar_position: 3
+image: og/wcs/user_guides.jpg
+---
+
+import Link from '@docusaurus/Link';
+
+Weaviate Embeddings is enabled by default at the organization level and is available to all Weaviate Cloud users. 
+
+## Authentication
+
+In order to use Weaviate Embeddings you only need a [connection to your Weaviate Cloud cluster](/developers/wcs/manage-clusters/connect). 
```

Comment:
Given that this is the administration page, do we need this section?

Reply:
I would personally leave it. This seems like an important topic some users might have questions about. Maybe I could come up with a new name for the page.

Reply:
Hmm. Administration imo makes sense for aspects related to managing it - e.g through the cloud console, whereas this is about client-side usage.

Imo "Quickstart" or "Choose a model" could be broadened out to "Usage" or so, and then this would fit in well there. 

Do we need an entire page dedicated to just the model selection for this service?